dockerfiles/bloodhound.Dockerfile

```diff
@@ -23,15 +23,18 @@ ARG AZUREHOUND_VERSION=v2.1.9
 ########
 # Builder init
 ################
-FROM --platform=$BUILDPLATFORM docker.io/library/node:20-alpine AS deps
+FROM --platform=$BUILDPLATFORM docker.io/library/node:20-alpine3.20 AS deps
```

Comment:
Did we want 3.23? 🤔

Reply:
3.20 is the highest version of Alpine right now